string bag_path
string rosbag_args
---
string out
